src/main/webapp/assets/js/settingMeeting.js

```diff
@@ -39,10 +41,35 @@ var SettingMeeting = function() {
 		dateField.val(value);
 	};
 
+	var settingMeetingValidate = function() {
+		$.validator.setDefaults({
+			errorClass: 'invalid',
+			validClass: "valid",
+			errorPlacement: function(error, element) {
+				$(element)
+					.closest("form")
+					.find("label[for='" + element.attr("id") + "']")
+					.attr('data-error', error.text());
+			},
+			submitHandler: function(form) {
```

Comment:
Is this function really necessary??, if yes, could we log something more specific like "all fields in settings meeting  form are correct" maybe?